filebeat/docs/fields.asciidoc

```diff
@@ -954,11 +970,19 @@ Contains fields for the Nginx access logs.
 
 
 [float]
+=== nginx.access.remote_ip_list
+
+type: array
+
+An array of remote IP addresses. It is a list because it is common to include, besides the client IP address, IP addresses from headers like `X-Forwarded-For`. See also the `remote_ip` field.
+
+
+[float]
 === nginx.access.remote_ip
 
 type: keyword
 
-Client IP address.
+Client IP address. The first non-private IP address from the `remote_ip_list` is preferred. If no public IP addresses are present, the first from the `remote_ip_list` array. 
```

Comment:
So if there are no public IP addresses in `remote_ip_list`, this field contains a private IP address, right? If that's true, we should be a bit more direct here and say something like: 

The first public IP address from the `remote_ip_list`. If no public IP addresses are present, this field contains the first private IP address from the `remote_ip_list` array.